lib/redcord/redis.rb

```diff
@@ -110,18 +130,30 @@ def find_by_attr(model, query_conditions, select_attrs=Set.new, index_attrs:, ra
       query_conditions: T::Hash[T.untyped, T.untyped],
       index_attrs: T::Array[Symbol],
       range_index_attrs: T::Array[Symbol],
+      custom_index_attrs: T::Array[Symbol],
       hash_tag: T.nilable(String),
+      index_name: T.nilable(Symbol),
     ).returns(Integer)
   end
-  def find_by_attr_count(model, query_conditions, index_attrs:, range_index_attrs:, hash_tag: nil)
+  def find_by_attr_count(
+        model,
+        query_conditions,
+        index_attrs:,
+        range_index_attrs:,
+        custom_index_attrs: Array.new,
+        hash_tag: nil,
+        index_name: nil
+      )
     Redcord::Base.trace(
       'redcord_redis_find_by_attr_count',
       model_name: model,
     ) do
+      index_name = index_name || :default
```

Comment:
can we use nil instead of the value "default"? nil will become "" once it gets to Lua

Reply:
agree